support-frontend/assets/helpers/abTests/abtestDefinitions.ts

```diff
@@ -122,4 +126,28 @@ export const tests: Tests = {
 			'^/uk/(checkout|thank-you)',
 		excludeContributionsOnlyCountries: true,
 	},
+	abNudgeToLowRegular: {
+		variants: [
+			{
+				id: 'control',
+			},
+			{
+				id: 'v1',
+			},
+			{
+				id: 'v2',
+			},
+		],
+		audiences: {
+			ALL: {
+				offset: 0,
+				size: 1,
+			},
+		},
+		isActive: false, // until we need to switch it on
+		referrerControlled: false, // requires a URL param to trigger
+		seed: 7, // needs to be different to other tests
+		targetPage: pageUrlRegexes.contributions.allGenericCheckouts,
```

Comment:
This will mean users who go to the recurring checkout via another route (e.g. direct from the landing page) will be put into this AB test, and will be included in the test analytics.
We can instead use the `persistPage` field here to only put users in this AB test if they've come via the one-time-checkout:

```
targetPage: oneTimeCheckoutOnlyRegex,
persistPage: genericCheckoutRegex,
```

Reply:
Ah, thank you - I missed that - updated to remove the allGenericCheckouts which isn't required too.